pygmt/datasets/venus_relief.py

```diff
@@ -0,0 +1,102 @@
+"""
+Function to download the Venus relief dataset from the GMT data server, and load as
+:class:`xarray.DataArray`.
+
+The grids are available in various resolutions.
+"""
+from typing import Literal
+
+from pygmt.datasets.load_remote_dataset import _load_remote_dataset
+from pygmt.helpers import kwargs_to_strings
+
+__doctest_skip__ = ["load_venus_relief"]
+
+
+@kwargs_to_strings(region="sequence")
+def load_venus_relief(
+    resolution="01d",
+    region=None,
+    registration: Literal["gridline", "pixel"] = "gridline",
+):
+    r"""
+    Load the Venus relief dataset in various resolutions.
+
+    .. figure:: https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_venus_relief.jpg
+       :width: 80%
+       :align: center
+
+       Venus relief dataset.
+
+    The grids are downloaded to a user data directory (usually
+    ``~/.gmt/server/venus/venus_relief/``) the first time you invoke this function.
+    Afterwards, it will load the grid from the data directory. So you'll need an
+    internet connection the first time around.
+
+    These grids can also be accessed by passing in the file name
+    **@venus_relief**\_\ *res*\[_\ *reg*] to any grid processing function or plotting
+    method. *res* is the grid resolution (see below), and *reg* is the grid registration
+    type (**p** for pixel registration or **g** for gridline registration).
+
+    The default color palette table (CPT) for this dataset is *@venus_relief.cpt*. It's
+    implicitly used when passing in the file name of the dataset to any grid plotting
+    method if no CPT is explicitly specified. When the dataset is loaded and plotted as
+    an :class:`xarray.DataArray` object, the default CPT is ignored, and GMT's default
+    CPT (*turbo*) is used. To use the dataset-specific CPT, you need to explicitly set
+    ``cmap="@venus_relief.cpt"``.
+
+    Refer to :gmt-datasets:`venus-relief.html` for more details about available
+    datasets, including version information and references.
+
+    Parameters
+    ----------
+    resolution : str
+        The grid resolution. The suffix ``d``, ``m`` and ``s`` stand for arc-degrees,
+        arc-minutes and arc-seconds. It can be ``"01d"``, ``"30m"``, ``"20m"``,
+        ``"15m"``, ``"10m"``, ``"06m"``, ``"05m"``, ``"04m"``, ``"03m"``, ``"02m"``, and
+        ``"01m"``.
```

Comment:
For the Venus relief dataset no grid resolution in arc-seconds are available.
```suggestion
        The grid resolution. The suffix ``d`` and ``m`` stand for arc-degrees and
        arc-minutes. It can be ``"01d"``, ``"30m"``, ``"20m"``, ``"15m"``, ``"10m"``,
        ``"06m"``, ``"05m"``, ``"04m"``, ``"03m"``, ``"02m"``, and ``"01m"``.
```